tests/Aspire.Playground.Tests/ProjectSpecificTests.cs

```diff
@@ -68,11 +68,11 @@ public async Task AzureFunctionsTest()
         await app.StartAsync();
         await app.WaitForResources().WaitAsync(TimeSpan.FromMinutes(2));
 
-        // Wait for the blobTrigger to be discovered as an indication that the host and worker
-        // has been successfully launched
+        // Wait for the 'Job host started' message as an indication
```

Comment:
Do you want to remove the ActiveIssue attribute?

Reply:
Done.